src/db/car/forest.rs

```diff
@@ -72,6 +72,8 @@ use std::{
 use tokio::io::{AsyncWrite, AsyncWriteExt};
 use tokio_util::codec::{Decoder, Encoder as _};
 
+pub const FOREST_CAR_FILE_EXTENSION: &str = ".forest.car.zst";
```

Comment:
Could we use this constant where we have other occurrences of this, e.g., `snapshot_cmd`?

Reply:
Yes, I will do this in a follow-up PR to not bloat this one

Reply:
https://github.com/ChainSafe/forest/pull/3427